tests/models/test_popular_in_category.py

```diff
@@ -444,3 +450,150 @@ def test_second_fit_refits_model(
             n_categories=n_categories,
         )
         assert_second_fit_refits_model(model, dataset)
+
+
+class TestPopularInCategoryModelConfiguration:
+    @pytest.fixture
+    def interactions_df(self) -> pd.DataFrame:
+        interactions_df = pd.DataFrame(
```

Comment:
Is it the same interactions, features and dataset that was used in the main test class? If so, let's move these fixtures  out from the class and reuse them

Reply:
Done.